.gitignore

```diff
@@ -15,13 +15,5 @@ npm-debug.log
 /bundles
 
 # ignore incline compiling
-/demo/**/*.js
-/demo/**/*.js.map
-/demo/**/*.d.ts
-/components/**/*.js
-/components/**/*.js.map
-/components/**/*.d.ts
-ng2-bootstrap.js
-ng2-bootstrap.d.ts
-ng2-bootstrap.js.map
+/ng2-bootstrap
```

Comment:
please revert this changes
demo - is a gh-pages actually
and all other
in any situation we should not commit
generated files